### ISSUE TITLE
SHM with microros-agent

### ISSUE THREAD

**Issue by aditya2592** [micro-ROS/docker#85]
Hi,

Both XML files in the [entrypoint](https://github.com/micro-ROS/docker/blob/humble/micro-ROS-Agent/micro-ros_entrypoint.sh) seem to be disabling SHM, is this actually needed? Shouldn't participants on same machine but in other docker containers still be able to talk to the agent using SHM?

**Comment by Acuadros95** [comment]
Hi @aditya2592,

The shared memory directory (`/dev/shm`) is not shared between containers, that is why we disable it by default.
To enable this communication, the directory should be mounted on your docker container: `-v /dev/shm:/dev/shm`.

**Comment by Acuadros95** [comment]
Yes, you have to modify the `FASTRTPS_DEFAULT_PROFILES_FILE` file of course.
